src/EA.Weee.Web/Areas/Admin/Controllers/HomeController.cs

```diff
@@ -75,10 +75,10 @@ public ActionResult ChooseActivity(RadioButtonStringCollectionViewModel viewMode
                     return RedirectToAction("ManageSchemes", "Scheme");
 
                 case InternalUserActivity.ManageAatfs:
-                    return RedirectToAction("ManageAatfs", "Aatf");
+                    return RedirectToAction("ManageAatfs", "Aatf", new { type = "AATF" });
 
                 case InternalUserActivity.ManageAes:
-                    return RedirectToAction("ManageAes", "Ae");
+                    return RedirectToAction("ManageAatfs", "Aatf", new { type = "AE" });
 
                 case InternalUserActivity.ProducerDetails:
```

Comment:
Can the unit test for this redirect check that the type route value is present and set to ae